agent/api/container/container_test.go

```diff
@@ -21,6 +21,9 @@ import (
 	"testing"
 	"time"
 
+	containerresource "github.com/aws/amazon-ecs-agent/agent/containerresource"
+	"github.com/aws/amazon-ecs-agent/agent/containerresource/containerstatus"
+
```

Comment:
nit: extra line here too and other files too

Reply:
Will update!